test/sha1.sh

```diff
@@ -15,8 +15,12 @@ declare s=(uint8:{1..20})
 
 # SHA_CTX is a typedef, not a struct, so you should use -a
 if ! sizeof -am ctx SHA_CTX; then
-    echo FIXME: failed to create SHA_CTX structure
-    exit 1
+    echo 'WARNING: failed to create SHA_CTX structure (missing debug symbols?) - falling back to manual definition'
+    # see SHAstate_st in <sha.h> for structure definition
+    if ! dlcall -r pointer -n ctx malloc $(($(sizeof unsigned) * (5 + 2 + 16 + 1))); then
```

Comment:
This seems a bit odd; openssl shouldn't have anything to do with the libc involved.  Any idea what went wrong?

Reply:
I wasn't sure at first, but eventually I found this which pointed me towards the answer: https://github.com/taviso/ctypes.sh/wiki/Structures#troubleshooting-automatic-structure

Thankfully, my distribution had an equivalent debug symbols package.. I just didn't notice.  It now works fine without this change, so I've removed it.  Thanks. :)